Rakefile

```diff
@@ -130,8 +130,11 @@ namespace :cordova do
   desc "Cordova build {platform}"
   task build: :prepare do
     Dir.chdir(CORDOVA_PATH) do
+      team_id = ENV['IOS_DEVELOPMENT_TEAM_ID']
+      provisioning_profile = ENV['PROVISIONING_PROFILE_STAGING']
+      provisioning_profile = ENV['PROVISIONING_PROFILE_PROD'] if(environment === "production")
       build = (environment == "staging" && platform == 'android') ? "debug" : "release"
-      system({"ENVIRONMENT" => environment}, "cordova compile #{platform} --#{build} --device")
+      system({"ENVIRONMENT" => environment}, "cordova compile #{platform} --#{build} --device --codeSignIdentity='iPhone Developer' --developmentTeam=#{team_id} --provisioningProfile=#{provisioning_profile}")
```

Comment:
Given we have the `codeSignIdentity`, `developmentTeam` and `provisionProfile` details in `cordova/build.json` do we need them here as well or can we just use `build.json` settings based on the environment selected - `debug` or `release`?

Reply:
@steveyken , yes using build.json would be ideal, and we can specify config file using this https://github.com/apache/cordova-ios/blob/master/bin/templates/scripts/cordova/build#L48 , 
But I tried it and got the error saying build.json file doesn't exist. 

Also tried creating one build.json and specifying the same configuration but still got the same error.

Also I noticed that build.json was using old provisioningProfile(we did not change it after last time we updated our pro. profiles), so if build.json was getting used during cordova build then none our build would've worked. So I suspect that cordova wasn't picking up configuration from build.json from the beginning.

And as upgrading to XCode was more important so I thought I should do that first and later we can figure the way to use build.json

Thanks